### PR TITLE
Add website title to site configuration access examples

### PR DESCRIPTION
**Pull request by schliesser**

### CODE REVIEW

File: Documentation/DataTypes/Properties/GetText.rst.txt [hunk 1]
```diff
@@ -634,6 +636,11 @@ siteLanguage
       
       page.10 = TEXT
 	   page.10.dataWrap = The current site language direction is {siteLanguage:direction}
+      
+      # Website title for the current language with fallback to the website title of the site configuration.
+      page.10 = TEXT
+      page.10.data = site:websiteTitle
```
Comment (gilbertsoft) [inline Documentation/DataTypes/Properties/GetText.rst.txt:642]: Please use the default fallback option instead like shown in Slack.

**Review by gilbertsoft — CHANGES_REQUESTED** [review]

File: Documentation/DataTypes/Properties/GetText.rst.txt [hunk 2]
```diff
@@ -634,6 +636,11 @@ siteLanguage
...
+      page.10.override.data = siteLanguage:websiteTitle
```
Comment (gilbertsoft) [inline Documentation/DataTypes/Properties/GetText.rst.txt]:
```suggestion
      page.10.data = siteLanguage:websiteTitle // site:websiteTitle
```

File: Documentation/DataTypes/Properties/GetText.rst.txt (hunk 1 above)
Comment (schliesser) [reply Documentation/DataTypes/Properties/GetText.rst.txt]: Done 👍

**Review by gilbertsoft — APPROVED** [review]
Thanks for this!

File: Documentation/DataTypes/Properties/GetText.rst.txt [hunk 3]
```diff
@@ -577,6 +577,7 @@ site
    :ts:`baseVariants`     The Base Variants for this site
    :ts:`rootPageId`       The Root Page ID of this site
    :ts:`identifier`       The identifier (name) of this site configuration
+   :ts:`websiteTitle`     The website title of this site configuration
```
Comment (gilbertsoft) [inline Documentation/DataTypes/Properties/GetText.rst.txt:580]:
```suggestion
   :ts:`websiteTitle`     The website title of this site
```

**Review by gilbertsoft — CHANGES_REQUESTED** [review]
Just one little thing

**Review by gilbertsoft — APPROVED** [review]

**Review by mbrodala — APPROVED** [review]

**Pull request merged**